src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -434,7 +435,7 @@ public void configure(ActionMenuWidget actionMenu, EntityBundle entityBundle, bo
 		this.entity = entityBundle.getEntity();
 		this.isUserAuthenticated = authenticationController.isLoggedIn();
 		this.isCurrentVersion = isCurrentVersion;
-		this.entityTypeDisplay = EntityTypeUtils.getDisplayName(EntityTypeUtils.getEntityTypeForClass(entityBundle.getEntity().getClass()));
+		this.entityTypeDisplay = getFriendlyEntityTypeName(entityBundle.getEntity());
```

Comment:
This changes 'Entity View Tools' to e.g. 'Project View Tools'

![image](https://user-images.githubusercontent.com/17580037/148288624-f01a46db-3bf8-44d2-8c8d-2e2a5c3b570e.png)